dotcom-rendering/src/web/components/KeyEventsContainer.tsx

```diff
@@ -5,17 +5,24 @@ import KeyEvents, {
 type Props = {
 	keyEvents: Block[];
 	format: ArticleFormat;
+	filterKeyEvents: boolean;
 };
 
-export const KeyEventsContainer = ({ keyEvents, format }: Props) => {
+export const KeyEventsContainer = ({
+	keyEvents,
+	format,
+	filterKeyEvents,
+}: Props) => {
 	const transformedKeyEvents: KeyEvent[] = keyEvents
 		.filter((keyEvent) => {
 			return keyEvent.title && keyEvent.blockFirstPublished;
 		})
 		.map((keyEvent) => {
 			return {
 				text: keyEvent.title || '', // We fallback to '' here purely to keep ts happy
-				url: `?page=with:block-${keyEvent.id}#block-${keyEvent.id}`,
+				url: `?filterKeyEvents=${
+					filterKeyEvents ? 'true' : 'false'
```

Comment:
as the original boolean is being interpolated, can we just use `filterKeyEvents` here rather than conditionally returning 'true' or 'false'?

Reply:
Yeah makes it a bit easier to read too. Updated.